spec/features/monitor/sign_in_out_spec.rb

```diff
@@ -0,0 +1,15 @@
+describe 'smoke test: sign in and out' do
+  let(:monitor) { MonitorHelper.new(self) }
+
+  before { monitor.setup }
+
+  it 'creates account, signs out, signs back in' do
+    visit monitor.idp_signup_url
+    creds = monitor.create_new_account_with_sms
```

Comment:
so I'm on the fence about leaving these all as methods on the `MonitorHelper` --- I like how they're explicit, I would like us to avoid mixing in all the same helpers we normally use in feature tests because those can depend on the DB

However, it does make it super verbose.... because they're no longer mixed in we have to do `context.visit ...` `context.expect()` ... etc

Reply:
Meh, I moved them and made them a normal helper that references `monitor.email_address` etc